extensions/rich_text_components/Math/protractor.js

```diff
@@ -20,9 +20,9 @@
 var objects = require(process.cwd() + '/extensions/objects/protractor.js');
 
 var customizeComponent = async function(modal, rawLatex) {
-  await objects.MathExpressionContentEditor(
-    modal.element(by.tagName('math-expression-content-editor'))
-  ).setValue(rawLatex);
+  await (objects.MathExpressionContentEditor(
+    await modal.element(by.tagName('math-expression-content-editor'))
```

Comment:
I don't think you need this inner `await`. `.element()` returns an ElementFinder, not a promise

Reply:
done, removed